Pipfile

```diff
@@ -19,7 +19,6 @@ gitignore-parser = "==0.1.1"
 kthread = "==0.2.3"
 markdown = "==3.4.4"
 marshmallow = "==3.20.1"
-modin = {extras = ["dask"], version = "==0.23.1"}
 networkx = "==2.6"
 openpyxl = "==3.1.2"
 pandas = "==2.0.0"
```

Comment:
we might want to update pandas as well since it's already at 2.1.4

Reply:
@florian-vuillemot Is working on upgrading dependencies. It should be part of his PR.